spec/workers/reactions/update_relevant_scores_worker_spec.rb

```diff
@@ -47,8 +47,10 @@
     end
 
     it "uses a throttled call for syncing the reactions count" do
-      allow(ThrottledCall).to receive(:perform)
```

Comment:
While this is not directly related to this PR I noticed this old spec and thought it would be better if it, you know, actually tested something.